### PR TITLE
[brand] Update jetpack.io -> jetify

### PR DESCRIPTION
**Pull request by mikeland73**
## Summary

Update jetpack.io to jetify.com. Will comment individually on backward compatibility.

### CODE REVIEW

File: envsec/internal/build/build.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func Issuer() string {
 }
 
 func Audience() string {
-	return "https://api.jetpack.io"
+	return "https://api.jetify.com"
```
Comment (mikeland73) [inline envsec/internal/build/build.go:36]: We are currently supporting both audiences so this is backwards compatible.

File: envsec/pkg/awsfed/awsfed.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func (a *AWSFed) AWSCredsWithLocalCache(
 	ctx context.Context,
 	tok *session.Token,
 ) (*types.Credentials, error) {
-	cache := filecache.New[*types.Credentials]("jetpack.io/envsec")
+	cache := filecache.New[*types.Credentials]("jetify.com/envsec")
```
Comment (mikeland73) [inline envsec/pkg/awsfed/awsfed.go:43]: This is cache, so no issues changing.

File: envsec/pkg/envsec/init.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ var (
 )
 
 const (
-	dirName       = ".jetpack.io"
+	dirName       = ".jetify.com"
```
Comment (mikeland73) [inline envsec/pkg/envsec/init.go:23]: This is gonna cause projects to be reinitialized, but that's OK.

File: pkg/runx/impl/httpcacher/defaults.go [hunk 4]
```diff
@@ -5,7 +5,7 @@ import (
 	"path/filepath"
 )
 
-const xdgSubdir = "jetpack.io/http"
+const xdgSubdir = "jetify.com/http"
```
Comment (mikeland73) [inline pkg/runx/impl/httpcacher/defaults.go:8]: This is for caching so OK

File: pkg/runx/impl/registry/registry.go [hunk 5]
```diff
@@ -12,7 +12,7 @@ import (
 	"go.jetpack.io/pkg/runx/impl/types"
 )
 
-var xdgInstallationSubdir = "jetpack.io/pkgs"
+var xdgInstallationSubdir = "jetify.com/pkgs"
```
Comment (mikeland73) [inline pkg/runx/impl/registry/registry.go:15]: This is for caching.

File: envsec/pkg/envsec/init.go (hunk 3 above)
Comment (LucilleH) [reply envsec/pkg/envsec/init.go:23]: Should we do `.jetify`?

File: pkg/runx/impl/httpcacher/defaults.go (hunk 4 above)
Comment (LucilleH) [reply pkg/runx/impl/httpcacher/defaults.go:8]: Should we do `jetify/http`?

File: pkg/runx/impl/registry/registry.go (hunk 5 above)
Comment (LucilleH) [reply pkg/runx/impl/registry/registry.go:15]: Should we do `jetify/pkgs`?

**Review by LucilleH — APPROVED** [review]
Pending QQs

File: envsec/internal/build/build.go (hunk 1 above)
Comment (mikeland73) [reply envsec/internal/build/build.go:36]: ory already has both

**Pull request merged**